keyboards/td2sk/fpad/post_rules.mk

```diff
@@ -0,0 +1,20 @@
+ifeq ($(strip $(ENCODER_ENABLE)), yes)
```

Comment:
as a heads up, this probably won't be accepted in it's current form, due to the custom/duplicated encoder code. 

Namely, there is a pending change to add support for custom encoder support, and that should (much) better handle this sort of scenario:
https://github.com/qmk/qmk_firmware/pull/21548

Reply:
I understand. I will fix this pull request after the custom encoder feature is merged into QMK.